### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by leesei**
Replace marked with brucedown
Fix screenshot link

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,15 +4,15 @@
 
 [![NPM](https://nodei.co/npm/morkdown.png?compact=true)](https://nodei.co/npm/morkdown/)
 
-Morkdown is primarily designed to render [GitHub Flavored Markdown](http://github.github.com/github-flavored-markdown/) (GFM), so it's ideal for your **README.md**. When rendering the Markdown, it uses the same syntax highlighter as GitHub (the Python [Pygments](http://pygments.org/) library) and the styling is near identical to GitHub. Markdown content is parsed using [marked](https://github.com/chjj/marked), a JavaScript Markdown parser capable of parsing GFM.
+Morkdown is primarily designed to render [GitHub Flavored Markdown](http://github.github.com/github-flavored-markdown/) (GFM), so it's ideal for your **README.md**. When rendering the Markdown, it uses the same syntax highlighter as GitHub (the Python [Pygments](http://pygments.org/) library) and the styling is near identical to GitHub. Markdown content is parsed using [brucedown](https://github.com/rvagg/node-brucedown/), a JavaScript Markdown parser capable of parsing GFM.
```
Comment (rvagg) [inline README.md]: I'd rather leave it as `marked` cause that's what `brucedown` uses. `brucedown` isn't a parser, it's just a bundling of other tools.

Comment (rvagg) [inline README.md]: could be "via brucedown" though

File: README.md [hunk 2]
```diff
@@ -41,13 +41,13 @@ Morkdown is packaged with the standard CodeMirror themes, you can switch to a di
...
+The default theme is **neat** but you can set your own default theme by saving a JSON file in your home directory named *`.morkdownrc`* with the following content: `{ "theme": "themename" }`. (My theme of choice is *monokai*).
 
 ## Installing & Using
 
 You'll need Google Chrome of course, plus you'll need a python interpreter on your system to get the syntax highlighting working&mdash;which should be fine for Linux and Mac users. Getting it running on Windows might be a little tricky (but presumably not impossible!).
 
-You can install from [npm](http://npmjs.org), the Node.js package manager, with <b><code>npm install -g morkdown</code></b> (you may need to `sudo` that depending on your setup). Once installed you can simply run the **`morkdown <path to file.md>`** command and you're away!
+You can install from [npm](http://npmjs.org), the Node.js package manager, with **`npm install -g morkdown`** (you may need to `sudo` that depending on your Node.js setup). Once installed you can simply run the **`morkdown <path to file.md>`** command and you're away!
```
Comment (rvagg) [inline README.md]: I'd rather leave `Node.js` out of this, it's dependent on system setup as well

**Comment by leesei** [comment]
@rvagg Updated.

**Comment by leesei** [comment]
@rvagg ping

**Comment by rvagg** [comment]
thanks a lot @leesei!

**Pull request merged**